src/SourceBuild/content/repo-projects/installer.proj

```diff
@@ -29,10 +29,12 @@
     <BuildArgs Condition="'$(TargetOS)' == 'freebsd' and '$(DotNetBuildSourceOnly)' == 'true'">$(BuildArgs) /p:PortableBuild=true</BuildArgs>
     <BuildArgs Condition="'$(TargetOS)' != 'windows'">$(BuildArgs) /p:CoreSetupRid=$(TargetRid)</BuildArgs>
 
-    <!-- Consume the source-built Core-Setup and toolset. This line must be removed to source-build CLI without source-building Core-Setup first. -->
-    <BuildArgs>$(BuildArgs) /p:PublicBaseURL=file:%2F%2F$(ArtifactsAssetsDir)</BuildArgs>
+    <BaseUrl>$(ArtifactsAssetsDir)</BaseUrl>
+    <BuildArgs>$(BuildArgs) /p:PublicBaseURL=file:%2F%2F$(BaseUrl)</BuildArgs>
```

Comment:
Why bring the BaseUrl property back?

Reply:
I had it refactored out in a previous commit when I was being more strict about hiding the produced assets so we'd only produce the PGO'd SDK archive and nothing else. Matt had me remove that, but I didn't re-inline the property.

Reply:
You will revert that change, right?

```suggestion
    <BuildArgs>$(BuildArgs) /p:PublicBaseURL=file:%2F%2F$(ArtifactsAssetsDir)</BuildArgs>
```